scripts/cnode-helper-scripts/guild-deploy.sh

```diff
@@ -371,7 +371,7 @@ download_cnodebins() {
   pushd "${HOME}"/tmp >/dev/null || err_exit
   echo -e "\n  Downloading Cardano Node archive created from GitHub.."
   rm -f cardano-node cardano-address
-  curl -m 200 -sfL https://github.com/intersectmbo/cardano-node/releases/download/${CARDANO_NODE_VERSION}/cardano-node-${CARDANO_NODE_VERSION}-linux.tar.gz -o cnode.tar.gz || err_exit " Could not download cardano-node release ${CARDANO_NODE_VERSION} from GitHub!"
+  curl -m 200 -sfL https://github.com/intersectmbo/cardano-node/releases/download/${CARDANO_NODE_VERSION}/cardano-node-${CARDANO_NODE_VERSION%-pre}-linux.tar.gz -o cnode.tar.gz || err_exit " Could not download cardano-node release ${CARDANO_NODE_VERSION} from GitHub!"
```

Comment:
I assume this got added in as an error, but even if so :

I dont think we want to entertain use of `-pre` (or any non-stable releases) in any of the core scripts that aim to merge to alpha/master unless they're in a very specific `if` condition , with a TODO comment to remove in future.

Reply:
This is not a mistake. The binaries/tarball does not contain the `-pre` portion of the pre-release name.

For example a release of `8.9.2` results in a URL of:
```
https://github.com/intersectmbo/cardano-node/releases/download/8.9.2/cardano-node-8.9.2-linux.tar.gz
```
However, for the pre-release of `8.10.1-pre` the URL is:
```
https://github.com/intersectmbo/cardano-node/releases/download/8.10.1-pre/cardano-node-8.10.1-linux.tar.gz
```
Without the `%-pre`  an error occurs trying to find the `cardano-node-8.10.1-pre-linux.tar.gz` tarball to download, which does not exist. I believe this is in preparation to move pre-releases to the full/final release. When no other commits/changes are merged the existing binaries could persist as the full release instead of rebuilding them from scratch only to apply a different name to the file.

Reply:
But as said earlier, we wouldn't want `-pre` as default in guild-deploy.sh, it is OK to check and add that only for sanchonet (against NETWORK), but other networks should use proper stable versions